pkg/client/src/app/pages/applications/analysis-wizard/review.tsx

```diff
@@ -108,7 +115,14 @@ export const Review: React.FunctionComponent<IReview> = ({
           </DescriptionListDescription>
         </DescriptionListGroup>
         <DescriptionListGroup>
-          <DescriptionListTerm>Included packages</DescriptionListTerm>
+          <DescriptionListTerm>
+            {
+              // t("wizard.terms.packages")
```

Comment:
Cleanup

Reply:
The comment is needed otherwise i18n removes the terms from the translation files.